### PR TITLE
I added a few functions related to SparkSQL

### PR DESCRIPTION
**Pull request by tokomakoma123**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(defproject yieldbot/flambo "0.8.1-SNAPSHOT"
+(defproject yieldbot/flambo "0.8.2-SNAPSHOT"
```
Comment (sorenmacbeth) [inline project.clj]: It's not necessary to bump the minor version here.

File: project.clj [hunk 2]
```diff
@@ -55,7 +55,7 @@
           :output-dir "doc/codox"
           :src-dir-uri "http://github.com/yieldbot/flambo/blob/develop/"
           :src-linenum-anchor-prefix "L"}
-  :javac-options ["-source" "1.7" "-target" "1.7"]
+  :javac-options ["-source" "1.8" "-target" "1.8"]
```
Comment (sorenmacbeth) [inline project.clj]: Do any of the functions you added require jvm 1.8?

File: src/clojure/flambo/api.clj [hunk 3]
```diff
@@ -77,6 +77,10 @@
   ([master app-name]
    (JavaSparkContext. master app-name)))
 
+;; added by MH
```
Comment (sorenmacbeth) [inline src/clojure/flambo/api.clj]: please remove

File: src/clojure/flambo/api.clj [hunk 4]
```diff
@@ -77,6 +77,10 @@
...
+(defn close [ctx]
```
Comment (sorenmacbeth) [inline src/clojure/flambo/api.clj]: you could probably use `memfn` here

File: src/clojure/flambo/sql.clj [hunk 5]
```diff
@@ -188,3 +206,22 @@
       (if (< i n)
         (recur (inc i) (conj! v (.get row i)))
         (persistent! v)))))
+
+
+;; by MH
```
Comment (sorenmacbeth) [inline src/clojure/flambo/sql.clj]: please remove

File: src/clojure/flambo/sql.clj [hunk 6]
```diff
@@ -188,3 +206,22 @@
...
+
+(defn show
+  ([df]   (.show df))
+  ([df n]   (.show df n)))
+
+(defn count [sql-ctx]
```
Comment (sorenmacbeth) [inline src/clojure/flambo/sql.clj]: memfn could be used here

File: src/clojure/flambo/sql.clj [hunk 7]
```diff
@@ -188,3 +206,22 @@
...
+  (.count sql-ctx))
+
+(defn create-global-temp-view [v-name]
```
Comment (sorenmacbeth) [inline src/clojure/flambo/sql.clj]: memfn

File: src/clojure/flambo/sql.clj [hunk 8]
```diff
@@ -188,3 +206,22 @@
...
+  (.createGlobalTempView v-name))
+
+(defn create-or-replace-temp-view [v-name]
```
Comment (sorenmacbeth) [inline src/clojure/flambo/sql.clj]: memfn

File: src/clojure/flambo/sql.clj [hunk 9]
```diff
@@ -3,15 +3,25 @@
...
+    (:import [org.apache.spark.api.java JavaSparkContext]
            [org.apache.spark.sql SQLContext Row Dataset Column]
            [org.apache.spark.sql.hive HiveContext]
-           [org.apache.spark.sql.expressions Window]))
+           [org.apache.spark.sql.expressions Window]
+
+           [org.apache.spark.sql.types DataType DataTypes StructField StructType
+            Metadata MetadataBuilder]
+           
+           [org.apache.spark.sql.types ArrayType BinaryType BooleanType
```
Comment (sorenmacbeth) [inline src/clojure/flambo/sql.clj]: Why are all these imports needed here?

**Comment by tokomakoma123** [comment]
@sorenmacbeth Great comments ! I've fixed my code according to the comments. 
I'll send a pull request after my tests are passed.

File: src/clojure/flambo/api.clj (hunk 3 above)
Comment (tokomakoma123) [inline src/clojure/flambo/api.clj]: OK

File: project.clj (hunk 1 above)
Comment (tokomakoma123) [inline project.clj]: I see.

File: project.clj (hunk 2 above)
Comment (tokomakoma123) [inline project.clj]: No.

File: src/clojure/flambo/api.clj (hunk 4 above)
Comment (tokomakoma123) [inline src/clojure/flambo/api.clj]: yes

File: src/clojure/flambo/sql.clj (hunk 9 above)
Comment (tokomakoma123) [inline src/clojure/flambo/sql.clj]: No, I have to remove them.

File: src/clojure/flambo/sql.clj (hunk 5 above)
Comment (tokomakoma123) [inline src/clojure/flambo/sql.clj]: Yes.

File: src/clojure/flambo/sql.clj (hunk 7 above)
Comment (tokomakoma123) [inline src/clojure/flambo/sql.clj]: Thanks !

File: src/clojure/flambo/sql.clj (hunk 8 above)
Comment (tokomakoma123) [inline src/clojure/flambo/sql.clj]: yes.

**Pull request merged**